tests/Equinox.EventStore.Integration/StoreIntegration.fs

```diff
@@ -457,7 +505,53 @@ type Tests(testOutputHelper) =
         test <@ [EsAct.AppendConflict; yield! sliceForward; EsAct.BatchForward] = capture.ExternalCalls @>
     }
 
-#if !STORE_MESSAGEDB
+#if STORE_MESSAGEDB
+    [<AutoData(SkipIfRequestedViaEnvironmentVariable="EQUINOX_INTEGRATION_SKIP_EVENTSTORE")>]
+    let ``Can combine snapshotting with caching against Store`` (ctx, skuId) = Async.RunSynchronously <| async {
```

Comment:
```suggestion
    let ``Can combine snapshotting with caching against MessageDb Store`` (ctx, skuId) = Async.RunSynchronously <| async {
```
or maybe pull out a `type MessageDbTests` in this file, or a `MessageDbIntegration.fs`

Reply:
If I pull out a separate file should I go all the way and copy all 6 tests into it?

Reply:
probably not (I hadnt thought this point through in full when raising) - in that breakdown of the snapshotting mechanisms, you have 
a) adjacent ones (mdb only for now, but could add to esdb)
b) embedded (esdb, sss)
c) unfolds based

Reply:
Seeing as we have RollingSnapshots being tested in this file already, I think we might want to keep these all in the same file. I'll pull it out into a separate `type`

Reply:
I've pulled out `GeneralTests` `RollingSnapshotsTest` and `AdjacentSnapshotTest` into their own types

Reply:
looks good